### ISSUE TITLE
Kebechet fails to generate changelog even though the difference of commit exists.

### ISSUE THREAD

**Issue by harshad16** [thoth-station/kebechet#917]
**Describe the bug**

Kebechet fails to generate changelog, as while executing the gather of tags or commit, it is getting empty string in return 
which causes the changelog difference to be 0.

**Kebechet: v1.6.5**
```
2021-11-18 04:46:00,372   1 INFO     kebechet:194: Running manager 'version' for 'thoth-station/invectio'
2021-11-18 04:46:01,471   1 INFO     kebechet.managers.version.version:417: Found an issue #122 which is a candidate for request of new version release: New minor release
2021-11-18 04:46:02,216   1 INFO     kebechet.managers.version.version:114: Old version found in sources: 0.1.0
2021-11-18 04:46:02,217   1 INFO     kebechet.managers.version.version:117: Computed new version: 0.2.0
2021-11-18 04:46:02,230   1 INFO     kebechet.managers.version.version:265: Computing changelog for new release from version '0.1.0' to version '0.2.0'
2021-11-18 04:46:02,238   1 INFO     kebechet.managers.version.version:279: Old version was not found in the git tag history, assuming initial release
2021-11-18 04:46:02,246   1 INFO     kebechet.managers.version.version:287: Smart Log : False
2021-11-18 04:46:02,255   1 INFO     kebechet.managers.version.version:306: Adding changelog to the CHANGELOG.md file
2021-11-18 04:46:02,264   1 INFO     kebechet.managers.version.version:338: Computed changelog has 0 entries
2021-11-18 04:46:02,265   1 INFO     kebechet.managers.version.version:483: Closing the issue as there is no changelog between the new release of thoth-station/invectio.
2021-11-18 04:46:03,259   1 INFO     kebechet:245: Finished management for 'thoth-station/invectio'
```
Reference: https://github.com/thoth-station/invectio/issues/122#issuecomment-972536220

**Comment by KPostOffice** [comment]
Sorry, this is a sneaky bug. I think this happened because we clone with depth=1 elsewhere. And because we are reusing the repo from before we don't download the full history.

**Comment by pacospace** [comment]
Also appeared in https://github.com/thoth-station/storages/issues/2481 and https://github.com/thoth-station/mi/issues/492

**Comment by xtuchyna** [comment]
Hello, kebechet released new version and tagged it on gh with `kebechet-` prefix, is that desired?
![image](https://user-images.githubusercontent.com/46632404/143072451-a9d1785a-646c-49d5-9fa7-26817a58b071.png)

**Comment by pacospace** [comment]
> Hello, kebechet released new version and tagged it on gh with `kebechet-` prefix, is that desired? ![image](https://user-images.githubusercontent.com/46632404/143072451-a9d1785a-646c-49d5-9fa7-26817a58b071.png)
> 
> previous tags don't have that

Thanks @xtuchyna! It was removed already in https://github.com/thoth-station/kebechet/pull/924, @KPostOffice should we make a new release for Kebechet to fix this meanwhile?

**Comment by harshad16** [comment]
This commit has fixed this issue: https://github.com/thoth-station/kebechet/commit/1dbd454e4837afed9503697ded8d4b9d421fef4d
also available in release v1.6.6 and above
